### PR TITLE
Dev-python/snakemake: version bump

### PR DESCRIPTION
**Pull request by AndrewAmmerlaan**

### CODE REVIEW

File: dev-python/datrie/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```
Comment (epsilon-0) [inline dev-python/datrie/metadata.xml:4]: not sure we can do this...
They need to agree with moving/copying to ::science
Or we can drop them as maintainer and only keep science.

File: dev-python/snakemake/snakemake-5.31.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# missing test dep google.cloud
+RESTRICT="test"
+
+BDEPEND="test? (
+	dev-python/requests-mock[${PYTHON_USEDEP}]
+	dev-python/google-api-python-client[${PYTHON_USEDEP}]
+	dev-python/jinja[${PYTHON_USEDEP}]
+	dev-python/networkx[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/pygraphviz[${PYTHON_USEDEP}]
```
Comment (epsilon-0) [inline dev-python/snakemake/snakemake-5.31.1.ebuild:28]: This feels very much like it is going to be doing network calls in the testing phase >.>

File: dev-python/toposort/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>mschu.dev@gmail.com</email>
+		<name>Michael Schubert</name>
+	</maintainer>
+	<maintainer type="project">
```
Comment (epsilon-0) [inline dev-python/toposort/metadata.xml:8]: Same, I don't think we should add python as maintainer.

File: dev-python/toposort/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
```
Comment (epsilon-0) [inline dev-python/toposort/metadata.xml:4]: can't add them

File: dev-python/datrie/metadata.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+		<email>mschu.dev@gmail.com</email>
+		<name>Michael Schubert</name>
+	</maintainer>
+	<maintainer type="project">
```
Comment (epsilon-0) [inline dev-python/datrie/metadata.xml:8]: can't add python

**Review by epsilon-0 — CHANGES_REQUESTED** [review]
starter comments.

File: dev-python/datrie/metadata.xml (hunk 1 above)
Comment (AndrewAmmerlaan) [reply dev-python/datrie/metadata.xml:4]: Oops, I thought I fixed that. Seems I messed up somewhere when copying things.

File: dev-python/snakemake/snakemake-5.31.1.ebuild (hunk 2 above)
Comment (AndrewAmmerlaan) [reply dev-python/snakemake/snakemake-5.31.1.ebuild:28]: Yes I think so too, which is why I'm not bothering with packaging the missing dependency, the testsuite will most likely fail anyway. I just added RESTRICT="test"

**Comment by AndrewAmmerlaan** [comment]
Metadata.xml files should be fine now :smile:

**Comment by epsilon-0** [comment]
Thanks a lot!

**Pull request closed without merging**